config/config/common-config.yaml

```
@@ -739,10 +743,29 @@ opa:
       limits:
         cpu: 500m
         memory: 300Mi
+    affinity: {}
+    tolerations: []
+    nodeSelector: {kubernetes.io/os: linux}
+
+    # Enable for audit pod to use cache instead of disk
+    writeToRAMDisk: false
 
-  # how often gatekeeper audits kubernetes resources, in seconds
+  # How often gatekeeper audits kubernetes resources, in seconds
   auditIntervalSeconds: 600
 
+  # A lower chunk size can reduce memory consumption of the auditing Pod
+  # but can increase the number of requests to the Kubernetes API server
+  auditChunkSize: 500
+
+  # Enable audit informer cache instead of requesting resources from Kubernetes API
+  auditFromCache: false
```

Wondering if we should structure it like this, it is a super nitpick think tho. 
```suggestion
  audit:
  	# How often gatekeeper audits kubernetes resources, in seconds
  	intervalSeconds: 600

  	# A lower chunk size can reduce memory consumption of the auditing Pod
  	# but can increase the number of requests to the Kubernetes API server
  	chunkSize: 500

  	# Enable audit informer cache instead of requesting resources from Kubernetes API
  	fromCache: false
```

---

I was considering this as well, but thought that it would be simpler to keep the names the same as in the upstream repo.